packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -42,6 +42,33 @@ class VMInfoCard extends StatelessWidget {
       ),
     );
   }
+
+  @override
+  Size get preferredSize {
+    if (table != null) {
+      return Size.infinite;
+    }
+    return Size.fromHeight(
+      areaPaneHeaderHeight +
+          (rowKeyValues?.length ?? 0) * defaultRowHeight +
+          defaultSpacing,
+    );
+  }
+}
```

Comment:
was this intended to be part of another PR? it seems unrelated to the rest of the changes

Reply:
It will be useful to have the VmInfoCard have a size, to use it directly in the object inspector and potentially other screens in the vm tools tab. I thought I would add that in this PR, since it is also a change to the VmInfoCard.